api/v1alpha1/cdapmaster_types.go

```diff
@@ -136,6 +141,10 @@ type CDAPServiceSpec struct {
 	AdditionalVolumeMounts []corev1.VolumeMount `json:"additionalVolumeMounts,omitempty"`
 	// SecurityContext overrides the security context for the service pods.
 	SecurityContext *SecurityContext `json:"securityContext,omitempty"`
+	// EnableSystemMetrics allows service to disable system metrics service which runs in a sidecar container
+	// by setting it to false. When property is not specified, taken as true
+	// SystemMetricsExporter spec should also be present in CR.
+	EnableSystemMetrics *bool `json:"enableSystemMetrics,omitempty"`
```

Comment:
nit: Add a little more info like - "Optional field that is considered along with CDAPMasterSpec.SystemMetricsExporter to start a metric collection container in services. SystemMetricsExporter is a global setting in the CDAPMasterSpec when present enables metric collection for all services. For overriding this value (prevent creating metric collection container) for individual services, this field can be set as false. No effect when SystemMetricsExporter is not present. "

Reply:
Added more info